test/school_test.rb

```diff
@@ -1,21 +1,24 @@
 require 'minitest/autorun'
 require 'minitest/pride'
-require './lib/school'
+require '../lib/school'
```

Comment:
this should not be modified, i think we talked about this in the codehelp channel